doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -48,32 +77,74 @@ described in the :ref:`dmclock-qos` section.
 mClock Profile Types
 ====================
 
-mclock profiles can be broadly classified into two types,
-
-- **Built-in**: Users can choose between the following built-in profile types:
-
-  - **high_client_ops** (*default*):
-    This profile allocates more reservation and limit to external-client ops
-    as compared to background recoveries and other internal clients within
-    Ceph. This profile is enabled by default.
-  - **high_recovery_ops**:
-    This profile allocates more reservation to background recoveries as
-    compared to external clients and other internal clients within Ceph. For
-    example, an admin may enable this profile temporarily to speed-up background
-    recoveries during non-peak hours.
-  - **balanced**:
-    This profile allocates equal reservation to client ops and background
-    recovery ops.
-
-- **Custom**: This profile gives users complete control over all the mclock
-  configuration parameters. Using this profile is not recommended without
-  a deep understanding of mclock and related Ceph-configuration options.
-
-.. note:: Across the built-in profiles, internal clients of mclock (for example
-          "scrub", "snap trim", and "pg deletion") are given slightly lower
-          reservations, but higher weight and no limit. This ensures that
-          these operations are able to complete quickly if there are no other
-          competing services.
+mclock profiles can be broadly classified into *built-in* and *custom* profiles,
+
+Built-in Profiles
+-----------------
+Users can choose between the following built-in profile types:
+
+high_client_ops (*default*)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This profile allocates more reservation and limit to external-client ops
+as compared to background recoveries and other internal clients within
+Ceph. This profile is enabled by default. The table shows the resource
+controls parameters set by the profile:
+
++------------------------+-------------+--------+-------+
+|  Service Type          | Reservation | Weight | Limit |
++========================+=============+========+=======+
+| client                 | 50%         | 2      | MAX   |
++------------------------+-------------+--------+-------+
+| background recovery    | 25%         | 1      | 100%  |
++------------------------+-------------+--------+-------+
+| background best-effort | 25%         | 1      | MAX   |
++------------------------+-------------+--------+-------+
+
+high_recovery_ops
+^^^^^^^^^^^^^^^^^
+This profile allocates more reservation to background recoveries as
+compared to external clients and other internal clients within Ceph. For
```

Comment:
the use of "internal clients" can be confusing to the end user, how about "other background operations within the OSD"?